UI/DraedonSummoning/CodebreakerUI.cs

```diff
@@ -68,14 +68,28 @@ public static float CommunicateButtonScale
             set;
         } = 1f;
 
+        public static float CancelButtonScale
+        {
+            get;
+            set;
+        } = 1f;
+
         public static float MechIconScale
         {
             get;
             set;
         } = 1f;
 
+        // This variable is currently permanently set to false due to being deemed unfinished and not fit for release.
+        // It used to be a local variable but has been moved to a property so that addon mods can easily enable it.
+        public static bool DraedonTalkFeatureEnabled
+        {
+            get;
+            set;
+        }
```

Comment:
If/when we use this UI in Calamity, I'm assuming we'll deprecate this property. Should we just make it so the getter always returns true and the setter no-ops?